### PR TITLE
Main refactoring of v6.3 diffraction release notes

### PR DESCRIPTION
**Pull request by sf1919**
**Description of work.**
The main refactoring the the Diffraction release notes ahead of release of 6.3. Includes:-

- Removing the developer warning text
- Adding extra subheadings
- Reordering  and grouping of notes where appropriate
- Rewording notes to improve consistency
- Adding links
- Spelling & Grammar check

Further changes will be made later when any separate files for v6.3 diffraction release notes are amalgamated into the final release notes.

**To test:**

- remove build/docs/doctrees
- build the documents again and check there are no warnings
- Check that the links work
- Check the information is still correct
- Grammar/Spelling check

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: docs/source/release/v6.3.0/diffraction.rst [hunk 1]
```diff
@@ -5,78 +5,96 @@ Diffraction Changes
...
+- Fixed a bug such that attenuation calculated in :ref:`AnvredCorrection <algm-AnvredCorrection>` is now accurate to within 0.5% for typical muR.
+- Restored behavior in :ref:`ConvertUnits <algm-ConvertUnits>` where negative ``TOF`` converts to negative ``d-Spacing`` when ``DIFA==0`` .
+- :ref:`LoadPDFgetNFile <algm-LoadPDFgetNFile>` now returns standard units for atomic distance rather than the label.
 - The integration range has been corrected inside :ref:`PDFFourierTransform v2 <algm-PDFFourierTransform-v2>`.
-- Fix problem with the create_vanadium action when running with tt_mode=Custom in the ISIS PEARL powder diffraction scripts. Create a separate Vanadium file for each different custom grouping file rather than one for all custom runs
+- :ref:`SaveFocusedXYE <algm-SaveFocusedXYE>` now correctly writes all spectra to a single file when ``SplitFiles`` is ``False``. Previously it wrote only a single spectrum.
+- For processing vanadium run it no longer finds the environment automatically in :ref:`SetSampleFromLogs <algm-SetSampleFromLogs>`.
```
Comment (DanielMurphy22) [inline docs/source/release/v6.3.0/diffraction.rst:46]: Not really sure how the PR https://github.com/mantidproject/mantid/pull/32548 relates to this release note! 
```suggestion
- :ref:`SetSampleFromLogs <algm-SetSampleFromLogs>` no longer finds the environment automatically, when processing vanadium runs .
```
Does that sound okay @peterfpeterson ?

File: docs/source/release/v6.3.0/diffraction.rst [hunk 2]
```diff
@@ -5,78 +5,96 @@ Diffraction Changes
...
+
+* A number of improvements have been made to the :ref:`Fitting tab <ui engineering fitting>` of the Engineering Diffraction interface
+
+  * Improved axes scaling in the plot
+  * Automatically disabled zoom and pan when opening the fit browser (as they interfered with the interactive peak adding tool).
+  * The plot is now made larger when undocked, unless the size of the overall interface has been expanded significantly
+  * The tab has been made more tolerant to users deleting or renaming the workspaces in the workbench Workspaces widget.
+
+* Updated the default values for :ref:`EnggEstimateFocussedBackground <algm-EnggEstimateFocussedBackground>` and in the fitting tab table to ``Niter = 50`` and ``XWindow = { 600 for TOF, 0.02 for d-Spacing }``.
+* The file filter in the Focus tab for calibration Region includes ``No Region Filter``, ``North``, ``South`` and now also ``Cropped``, ``Custom``, ``Texture`` and ``Both Banks``. The text for ``No Unit/Region Filter`` are colored grey.
```
Comment (DanielMurphy22) [inline docs/source/release/v6.3.0/diffraction.rst:76]:
```suggestion
* The file filter in the Focus tab for calibration Region includes ``No Region Filter``, ``North``, ``South`` and now also ``Cropped``, ``Custom``, ``Texture`` and ``Both Banks``. The text for ``No Unit/Region Filter`` is colored grey.
```
This looks like my PR! Sorry!

File: docs/source/release/v6.3.0/diffraction.rst [hunk 3]
```diff
@@ -5,78 +5,96 @@ Diffraction Changes
...
+############
+- ``GetDetOffsetsMultiPeaks``, which is deprecate since v6.2.0, is removed.
+- ``CalibrateRectangularDetectors``, which is deprecate since v6.2.0, is removed.
 
 Engineering Diffraction
 -----------------------
 New features
 ############
-- Now support two texture grouping schemes: Texture20 (10 groups per bank, 20 in total) and Texture30 (15 groups per bank, 30 in total) for ENGIN-X in the :ref:`Engineering Diffraction interface<Engineering_Diffraction-ref>`. Note this involved changes to the bankID log values saved with focused data, so this means the UI will not load in previously focused .nxs files.
+- Now supports two texture grouping schemes: ``Texture20`` (10 groups per bank, 20 in total) and ``Texture30`` (15 groups per bank, 30 in total) for ``ENGIN-X`` in the :ref:`Engineering Diffraction interface<Engineering_Diffraction-ref>`. Note this involved changes to the ``bankID`` log values saved with focused data, so this means the UI will not load in previously focused ``.nxs`` files.
```
Comment (DanielMurphy22) [inline docs/source/release/v6.3.0/diffraction.rst:59]: This looks like the most impressive thing on this page! Here's an image from @RichardWaiteSTFC for this note
TextureGroup20
.

**Review by DanielMurphy22 — CHANGES_REQUESTED** [review]
Super good job! Just a few nitpicks and an image!

File: docs/source/release/v6.3.0/diffraction.rst (hunk 1 above)
Comment (sf1919) [reply docs/source/release/v6.3.0/diffraction.rst:46]: I've checked and this came from PR #32537 . @Kvieta1990 can you help with this?

Comment (peterfpeterson) [reply docs/source/release/v6.3.0/diffraction.rst:46]: How about
```rst
Add option to enable (default on) finding the sample environment automatically using :ref:`SetSampleFromLogs <algm-SetSampleFromLogs>`. This is used to turn off the feature for vanadium measurements when using :ref:`mantid.utils.absorptioncorrutils`
```

**Review by DanielMurphy22 — APPROVED** [review]
Thank you for the effort in particular for the Diffraction release notes! They look really good!

**Pull request merged**